module/Accounting/src/Accounting/Controller/OrganizationStatementController.php

```diff
@@ -26,11 +30,17 @@ class OrganizationStatementController extends OrganizationAwareController
 	 * @var Acl
 	 */
 	private $acl;
-
+	/**
+	 *
+	 * @var integer
+	 */
+	protected $pageSize;
```

Comment:
Nota stilistica:
1. definisci una costante di classe `DEFAULT_TRANSACTIONS_LIMIT = 10`
2. imposti `$pageSize = self::DEFAULT_TRANSACTIONS_LIMIT` alla dichiarazione della variabile e non nel costruttore (non è significativo nel costruttore)
3. coerentemente ai miei commenti precedenti rinomina `$pageSize` in `$transactionsLimit`